src/rard/research/migrations/0070_add_duplicates_field.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 3.2 on 2024-03-07 12:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("research", "0069_anonymousfragment_anonymous_fragments"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="anonymousfragment",
+            name="duplicate_frags",
+            field=models.ManyToManyField(
+                blank=True,
+                related_name="anonymousfragment_duplicate_fragments",
+                to="research.Fragment",
+            ),
+        ),
+        migrations.AddField(
+            model_name="fragment",
+            name="duplicate_frags",
+            field=models.ManyToManyField(
+                blank=True,
+                related_name="fragment_duplicate_fragments",
+                to="research.Fragment",
+            ),
+        ),
+        migrations.AddField(
+            model_name="testimonium",
+            name="duplicate_frags",
+            field=models.ManyToManyField(
+                blank=True,
+                related_name="testimonium_duplicate_fragments",
+                to="research.Fragment",
+            ),
+        ),
+        migrations.AddField(
+            model_name="anonymousfragment",
+            name="duplicate_afs",
+            field=models.ManyToManyField(
+                blank=True,
+                related_name="anonymousfragment_duplicate_anonfragments",
+                to="research.AnonymousFragment",
+            ),
+        ),
+        migrations.AddField(
+            model_name="fragment",
+            name="duplicate_afs",
+            field=models.ManyToManyField(
+                blank=True,
+                related_name="fragment_duplicate_anonfragments",
+                to="research.AnonymousFragment",
+            ),
+        ),
+        migrations.AddField(
+            model_name="testimonium",
```

Comment:
I'm not sure we did ask if we should enable it for them, I'll make note to do that. They are currently redundant but are there because it's on the `HistoricalBaseModel` and it complained when I removed this from the migration.